tasks/install-on-Windows.yml

```diff
@@ -1,16 +1,16 @@
 ---
 - name: make password for user
   set_fact:
-    buildkite_agent_user_password: "{{ lookup('password', '/tmp/bk-agent-password length=32 chars=ascii_letters,digits,punctuation') }}"
-  no_log: "{{ buildkite_agent_hide_secrets | default(true) }}"
+    buildkite_agent_user_password: '{{ lookup(''password'', ''/tmp/bk-agent-password length=32 chars=ascii_letters,digits,punctuation'') }}'
```

Comment:
This line seems iffy to me - the double single-quotes seem wrong.

Does this pass lint and work?

Reply:
This is the YAML way to escape `'`. Even though one might think `\'` would be the way to do it, it's not. [Reference](https://yaml.org/spec/current.html#id2534365)